geometry_msgs/Pose2D pose2d
uint16 intr_pnt_brd
---
uint16 intr_pnt_graph
